R/scholar.r

```diff
@@ -252,3 +252,86 @@ plot_coauthors <- function(network, size_labels = 5) {
     ggplot2::labs(title = paste0("Network of coauthorship of ", network$author[1])) +
     ggraph::theme_graph(title_size = 16, base_family = "sans")
 }
+
+
+
+
+
+
+
+
+##' Get author order.
+##'
+##' Get author rank in authors list.
+##'
+##' @examples
+##' library(scholar)
+##'
+##' id <- "bg0BZ-QAAAAJ&hl"
+##'
+##' publications <- scholar::get_publications(id)
+##' publications <- publications$author
+##' author <- scholar::get_profile(id)
+##' author <- author$name
```

Comment:
I would just use author$name directly in the function call

Reply:
Corrected